index.html

```diff
@@ -126,7 +126,58 @@
+                                                <li>Statyków jest więcej. Bo przecież klasa wykonująca <a href="https://github.com/laravel/framework/blob/v5.0.32/src/Illuminate/Support/Str.php">operacje na ciągach znaków</a> jest tylko kontenerem do przechowywania funkcji.</li>
+                                        </ol>
+                                <p>
+                                        Nadużywanie metod magicznych:
+                                </p>
+                                        <ol>
+                                                <li>Wspomniane wyżej połączenie dwóch query builderów.</li>
+                                                <li><a href="https://github.com/laravel/framework/blob/v5.0.32/src/Illuminate/Auth/AuthManager.php">AuthManager</a> również pozwala nam wykonywać <a href="https://github.com/laravel/framework/blob/v5.0.32/src/Illuminate/Support/Manager.php#L137">zgoła nieoczekiwane operacje</a> na <a href="http://laravel.com/docs/5.0/authentication#retrieving-the-authenticated-user">obiektach</a>, o których nie mamy większego pojęcia. <a href="">Dyskusja</a> (dotyczyła wersji 4.2, ale wygląda na to, że nadal jest aktualna)</li>
+                                                <li>Mockowanie tych szalonych zależności wymaga intensywnego korzystania z xdebuga.</li>
+
+                                        </ol>
+                                <p>
+                                        Występują też w przyrodzie zjawiska, których po prostu nie rozumiem:
+                                </p>
+                                        <ol>
+                                                <li>Packagist laravelish enough is not! W związku z czym istenieje <a href="http://packalyst.com/">Packalyst</a></li>
+                                        </ol>
+                                <p>
+                                Dlaczego w takim razie ktoś w ogóle używa Laravela? Z bardzo prostej przyczyny - Otwell zapewnia projektowi fenomenalny wręcz marketing, próg wejścia jest w zasadzie zerowy (najwyżej opakujesz kod proceduralny w statyczne metody), a do tego czas nas czasem goni... Co zrobić? Jak żyć? Nie używać Laravela do żadnych projektów, które potem będą musiały być przez dłuższy czas utrzymywane. Twoim celem jest zwiększenie sprzedaży piwa marki X w wakacje, wśród studentów? Zastanów się nad tym, czy nie byłoby warto użyć <a href="http://silex.sensiolabs.org/">Silexa</a> albo <a href="http://www.slimframework.com/">Slima</a>, a jeśli nie chcesz bądź po prostu musisz - to piszże w Laravelu.<br>
+                                Nigdy, przenigdy, nie używaj Laravela do projektów, które będą miały być utrzymywane przez wiele lat. Słaba architektura Cię wykończy zanim zdąży Cię wykończyć brak dobrych testów jednostkowych.
```

Comment:
No właśnie, tylko że FAQ PHPers to raczej nie są doświadczenia i wnioski jednej osoby a raczej całego community. Poza tym ja się z tym stwierdzeniem nie zgadzam, framework to detal a detale nie mogą wyłożyć całego projektu.